src/grt/README.md

```diff
@@ -127,6 +129,12 @@ antennas.  It uses the  `antennachecker` tool to identify any nets with antenna
 violations and, for each such net, the exact number of diodes necessary to fix the
 antenna violation.
 
+Options description:
+
+- `-macro_halo_x`. Specify the horizontal halo size (in microns) around macros.
+- `-macro_halo_y`. Specify the vertical halo size (in microns) around macros.
+Both options above should be the same defined during the floorplanning.
```

Comment:
If this is true, wouldn't it be easier to set the halo on the macros and use that from the database instead of adding additional parameters? What happens if we want to have different halos around different macros?

Reply:
I don't think we have this information in the database right now. This halo I'm talking about is defined during the cut rows process of tapcell. If there is a way to store this in the database, I agree with you that it would be easier than adding parameters. But it is not easy to detect these halos in the current state.

Reply:
I was trying to avoid iterating through all the rows for each diode since it could be expensive in runtime. But I'll try this approach and check how bad it is.